Don't use gimp-remote since gimp 2.4 already includes gimp-remote functionality
--- a/ufraw_conf.c
+++ b/ufraw_conf.c
@@ -137,7 +137,7 @@
 #elif HAVE_GIMP_2_4
     "gimp", /* remoteGimpCommand */
 #else
-    "gimp-remote", /* remoteGimpCommand */
+    "gimp", /* remoteGimpCommand */
 #endif
 
     /* EXIF data */
